esx_service/vmdk_ops.py

```diff
@@ -668,20 +670,27 @@ def vm_uuid2name(vm_uuid):
         return None
     return vm.config.name
 
-# Return error, or None for OK.
-def attachVMDK(vmdk_path, vm_uuid):
-    vm = findVmByUuid(vm_uuid)
-    logging.info("*** attachVMDK: %s to %s VM uuid = %s",
-                 vmdk_path, vm.config.name, vm_uuid)
-    return disk_attach(vmdk_path, vm)
 
+def attach_or_detach_VMDK(cmd, vmdk_path, vm_name, bios_uuid, vc_uuid):
```

Comment:
@shuklanirdesh82 
> this is confusing, #1526 (comment) "Testing done" section does say test-esx ran.

yeah. forgot to commit. That will teach me to not work at night. Stand by for a refresh


@shaominchen 
> ideally a function should do one thing and only one thing. 

Well, no. Code by definition does different things ,otherwise we end up with no code reuse. besides, attach/detach operation is one thing and only one thing., at least in my eyes and the code for both is identical. 
I understand your  taste preference is different, but I see it as a taste preference